--- odoo-custom/price_paper/models/customer_product_price.py	2020-03-02 15:54:06.000000000 -0500
+++ odoo-custom/price_paper/models/customer_product_price.py	2020-09-17 12:37:04.598443299 -0400
@@ -20,15 +20,15 @@
     lock_expiry_date = fields.Date(string='Lock Expiry date')
 
 
-    @api.multi
-    def write(self, vals):
-        """
-        overriden to update price_last_updated
-        """
-        result = super(CustomerProductPrice, self).write(vals)
-        if vals.get('price'):
-            self.price_last_updated = date.today()
-        return result
+#    @api.multi
+#    def write(self, vals):
+#        """
+#        overriden to update price_last_updated
+#        """
+#        result = super(CustomerProductPrice, self).write(vals)
+#        if vals.get('price'):
+#            self.price_last_updated = date.today()
+#        return result
 
     @api.constrains('partner_id')
     def check_partner(self):
